cmd/fluent-manager/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM --platform=$BUILDPLATFORM golang:1.21.5-alpine3.17 as builder
+FROM --platform=$BUILDPLATFORM golang:1.22.5-alpine3.20 as builder
 
```

Comment:
Why is it 1.22.5 and not 1.22.0?

Reply:
Go released 1.22.5 recently, it might be ok?

Reply:
I wiil change it